lib/logstash/filters/aggregate.rb

```diff
@@ -195,6 +195,30 @@ class LogStash::Filters::Aggregate < LogStash::Filters::Base
   # Example value : `"map['sql_duration'] += event['duration']"`
   config :code, :validate => :string, :required => true
 
+
+
+  # The code to execute to complete timeout generated event, when 'push_map_as_event_on_timeout' or 'push_previous_map_as_event' is set to true. 
+  # The code block will have access to the newly generated timeout event that is pre-populated with the aggregation_map values. 
+  #
+  # If 'timeout_task_id_field' is set, the event is also populated with the task_id value 
+  #
+  # Example value: `"event['tags'] = '_aggregatetimeout'"`
+  config :timeout_code, :validate => :string, :required => false
+
+
+  # This option indicates the timeout generated event's field for the "task_id" value. 
+  # The task id will then be set into the timeout event. This can help correlate which tasks have been timed out.  
+  #
+  # This field has no default value and will not be set on the event if not configured.
+  #
+  # Example:
+  #
+  # If the taskId is "12345" and this field is set to "my_Id", the generated event will have:
+  # event[ "my_Id" ] = "12345"
+  #
```

Comment:
Just a last thing about this description : 
could you replace "taskId" by "task_id" and "my_Id" by "my_id" ?